src/plugins/test_runner/src/TestRunnerPlugin.cpp

```diff
@@ -38,7 +38,7 @@
 
 #define SETTINGS_ROOT QString("test_runner/")
 /** Default number of threads to run XML tests. */
-#define NUM_THREADS_VAR_VALUE "1"
+#define NUM_THREADS_VAR_VALUE "10"
```

Comment:
This magic number is not good: some (virtual) hosts may have 1 CPU, others (real) hosts may have 16 real CPUs and 10 threads is not good for either of them.

Auto-detection won't help too, because you do not know a user intention.

A better and a simpler solution would be keeping "1" by default and allowing override with any user value using environment variable.

Reply:
Fixed